dspace-server-webapp/src/main/java/org/dspace/app/rest/utils/Utils.java

```diff
@@ -93,6 +92,11 @@
 
     private static final int EMBEDDED_PAGE_SIZE = 20;
 
+    /**
+     * The maximum number of embed levels to allow.
+     */
+    private static final int EMBED_MAX_LEVELS = 2;
```

Comment:
Yes, it occurred to me that this could be a server-wide setting or even a per-request arg. If there are use cases for that, it shouldn't be hard to add. We should still probably enforce a maximum, though.

Reply:
I had the same thought on configurability, though we may want to put a lot of warnings around that or enforce a maximum.  In any case though, I honestly feel that could come in a future PR, rather than trying to "perfect" everything in one PR.